code/modules/mob/living/brain/posibrain.dm

```diff
@@ -57,6 +67,16 @@ GLOBAL_VAR(posibrain_notify_cooldown)
 	update_icon()
 	addtimer(CALLBACK(src, .proc/check_success), askDelay)
 
+/obj/item/mmi/posibrain/AltClick(mob/living/user)
+	if(!istype(user) || !user.canUseTopic(src, BE_CLOSE))
+		return
+	var/input_seed = stripped_input(user, "Enter a personality seed", "Enter seed", ask_role)
```

Comment:
is there a max string lenght for this

Reply:
The default is 1024, but on second thought it's probably too much

Reply:
just make it name length which is ~48.

"Paladin AI" / "Malf AI" / "AI with custom laws"/ "gygax with weapons"

We also don't want to spam the popupboxes / chat with absurdly long messages.